src/gluonnlp/models/transformer.py

```diff
@@ -196,7 +196,7 @@ def __init__(self,
                                        bias_initializer=bias_initializer,
                                        dtype=self._dtype)
         attention_layout = 'NTK' if self._layout == 'NT' else 'TNK'
-        self.attention_cell =\
+        self.self_attention =\
```

Comment:
You may keep the `attention_cell` since there are existing models that use this `TransformerEncoderLayer`.